### PR TITLE
Added --input-index and --output-index flags

### PR DESCRIPTION
**Pull request by kjvalencik**
Fixes https://github.com/taskrabbit/elasticsearch-dump/issues/121

### CODE REVIEW

**Comment by kjvalencik** [comment]
@evantahler I did my best to match the style, but let me know if you'd like to change something.

**Comment by kjvalencik** [comment]
That failing test is a bit flaky. I think it might be missing a `_refresh`.

File: README.md [hunk 1]
```diff
@@ -94,8 +102,12 @@ Usage: elasticdump --input [SOURCE] --output [DESTINATION] [OPTIONS]
 
 --input                       
                     Source location (required)
+--input-index
```
Comment (evantahler) [inline README.md]: What are the defaults here?

File: README.md [hunk 2]
```diff
@@ -190,6 +202,7 @@ NOTE: only works for output
 - this tool is likley to require Elasticsearch version 1.0.0 or higher
 - elasticdump (and elasticsearch in general) will create indices if they don't exist upon import
 - when exporting from elasticsearch, you can have export an entire index (`--input="http://localhost:9200/index"`) or a type of object from that index (`--input="http://localhost:9200/index/type"`).  This requires ElasticSearch 1.2.0 or higher
+- If elasticsearch is in a sub-directory, index and type must be provided with a separate argument (`--input="http://localhost:9200 --input-index=index/type"`). Using `--input-index=/` will include all indices and types.
```
Comment (evantahler) [inline README.md]: You should show what a "standard" install looks like, and also a "custom" one, and how you would use this flag

**Comment by evantahler** [comment]
Looks really good (great tests)!
I've added a few notes on the documentation, so make those changes, and I'll merge this in!

**Comment by kjvalencik** [comment]
@evantahler Added additional documentation and examples for using the flag and when it is required: https://github.com/taskrabbit/elasticsearch-dump/pull/128/files#diff-04c6e90faac2675aa89e2176d2eec7d8R92

**Comment by evantahler** [comment]
:+1: 
Thanks!

**Comment by evantahler** [comment]
> npm publish
+ elasticdump@0.16.0

**Pull request merged**